src/seedu/addressbook/data/AddressBook.java

```diff
@@ -57,9 +61,12 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
     /**
      * Clears all persons and tags from the address book.
      */
-    public void clear() {
-        allPersons.clear();
-    }
+    public void clear() {allPersons.clear();}
```

Comment:
Can follow previous layout

Reply:
It's better to observe code standard here to change line with brackets